### ISSUE TITLE
Inconsistent schedule and status

### ISSUE THREAD

**Issue by khuldraeseth** [CarsonHoffman/office-hours-queue#35]
2014efc4c23de95d7924d31b6eefdeb4d55eee03 broke things. I have my browser reporting UTC as its local time. Translating in my head would be easy enough, but the site shows inconsistent times:
![Inconsistent times. This image shows what Unix `date` reports, what in-browser `Date()` reports, and the OH queue site telling me both 2:00 and 3:00 for the time the queue opens.](https://imgur.com/auWsswV.png)
In the top right, I'm told the queue opens at 2:00 PM. Minus four hours and you get 10:00 AM, which is the time the queue did actually open a couple minutes ago. The schedule shows me 3:00 PM instead.

Likely related is the site reporting at noon two days ago that the queue was both open and closed simultaneously:
![Inconsistent status. Top right shows that the queue is open until 9:30, but where the queue would normally be is a blue box saying the queue is closed.](https://imgur.com/aUgDsyO.png)
This screenshot also shows an unfamiliar button layout, with just "Open Queue" in place of "Edit Schedule" and "Clear Queue". Clicking that button did open the queue immediately and seemingly did not change the schedule.

**Comment by CarsonHoffman** [comment]
I don't think that [2014efc](https://github.com/CarsonHoffman/office-hours-queue/commit/2014efc4c23de95d7924d31b6eefdeb4d55eee03) has anything to do with this (that change did nothing to change how the schedule screen you're showing is rendered), but it does seem that I wasn't as careful as I should have been around daylight saving time. I just pushed and deployed [0741ba8](https://github.com/CarsonHoffman/office-hours-queue/commit/0741ba861edba8e27888dacefcd644c22ecdf3a3); see if that fixes things.
